pyproject.toml

```
@@ -94,7 +94,7 @@ pyvista = [
 [tool.black]
 line-length = 100
 skip-string-normalization = true
-target-version = ['py39']
+target-version = ['py38']
```

This can be reverted, I don't think it was intentional and I may have confused it with the target version for `ruff`.
```suggestion
target-version = ['py39']
```